openlibrary/templates/lib/nav_foot.html

```diff
@@ -6,40 +6,40 @@
-          <li><a href="https://blog.openlibrary.org/" title="$_('Blog')">Blog</a></li>
-          <li><a href="https://archive.org/about/terms.php" title="$_('Terms of Service')">Terms of Service</a></li>
-          <li><a href="https://archive.org/donate/?platform=ol" title="$_('Donate')">Donate</a></li>
+          <li><a href="/about/vision" title="$_('Vision')">$_('Vision')</a></li>
+          <li><a href="https://archive.org/about/jobs.php" title="$_('Jobs')">$_('Careers')</a></li>
+          <li><a href="https://blog.openlibrary.org/" title="$_('Blog')">$_('Blog')</a></li>
+          <li><a href="https://archive.org/about/terms.php" title="$_('Terms of Service')">$_('Terms of Service')</a></li>
+          <li><a href="https://archive.org/donate/?platform=ol" title="$_('Donate')">$_('Donate')</a></li>
         </ul>
       </div>
       <div>
-        <h2>Discover</h2>
+        <h2>$:_('Discover')</h2>
         <ul>
-          <li><a href="/" title="$_('Go home')">Home</a></li>
-          <li><a href="/search" title="$_('Explore Books')">Books</a></li>
-          <li><a href="/search/authors" title="$_('Explore authors')">Authors</a></li>
-          <li><a href="/subjects" title="$_('Explore subjects')">Subjects</a></li>
-          <li><a href="/advancedsearch" title="$_('Advanced Search')">Advanced Search</a></li>
-          <li><a href="#top" title="$_('Navigate to top of this page')">Return to Top</a></li>
+          <li><a href="/" title="$_('Go home')">$_('Home')</a></li>
+          <li><a href="/search" title="$_('Explore Books')">$_('Books')</a></li>
+          <li><a href="/search/authors" title="$_('Explore authors')">$_('Authors')</a></li>
+          <li><a href="/subjects" title="$_('Explore subjects')">$_('Subjects')</a></li>
+          <li><a href="/advancedsearch" title="$_('Advanced Search')">$_('Advanced Search')</a></li>
+          <li><a href="#top" title="$_('Navigate to top of this page')">$_('Return to Top')</a></li>
         </ul>
       </div>
       <div>
-        <h2>Develop</h2>
+        <h2>$:_('Develop')</h2>
         <ul>
-          <li><a href="/developers" title="$_('Explore Open Library Development Center')">Development Center</a></li>
-          <li><a href="/developers/api" title="$_('Explore Open Library APIs')">API Documentation</a></li>
-          <li><a href="/developers/dumps" title="$_('Bulk Open Library data')">Bulk Data Dumps</a></li>
+          <li><a href="/developers" title="$_('Explore Open Library Development Center')">$_('Development Center')</a></li>
+          <li><a href="/developers/api" title="$_('Explore Open Library APIs')">$_('API Documentation')</a></li>
+          <li><a href="/developers/dumps" title="$_('Bulk Open Library data')">$_('Bulk Data Dumps')</a></li>
           <li><a href="https://github.com/internetarchive/openlibrary/wiki/Writing-Bots" title="$_('Write a bot')">Writing Bots</a></li>
```

Comment:
Line below needs "Writing Bots" translated.

Reply:
Added the missing line. Thank you.